crates/sui-e2e-tests/tests/bridge_tests.rs

```diff
@@ -46,6 +56,139 @@ async fn test_create_bridge_state_object() {
     }
 }
 
+#[tokio::test]
+async fn test_committee_registration() {
```

Comment:
can we add a test that shows that the registration is run only once? 
and do we know what happens if it runs more than once?

Reply:
there are another check in move to prevent init_committee from running twice

Reply:
I saw it but that cannot be the way we are covered, or is it?
I mean I would expect the committee transaction to be created only once, no?